index.html

```diff
@@ -70,6 +70,215 @@ <h1>
         </button>
       </div>
     </section>
+    <section id="works">
+      <h2 class="title">
+        My Recent Works
+      </h2>
+      <hr>
+      <div id="grid">
+        <div class="card-container">
+          <div class="card-image">
+            
+          </div>
```

Comment:
- [ ] Please use an `img` tag with an image of grey background instead of making a `div` tag. :+1: Please implement this on all cards :blush:

Reply:
Hello, @HAMMAS-SALEEM If you see the Figma of this design I think there is no kind of image with a bg. what do you?

Reply:
Yes, you are right. :+1: 
But it's a place of the image. This is a template for us to create a basic portfolio. But we shall need to add images in these cards when we will have projects. :thinking: That's why it should be an `img` tag.